templates/clients.html

```diff
@@ -68,7 +68,7 @@ <h4 class="modal-title">QR Code</h4>
                     <span aria-hidden="true">&times;</span>
                 </button>
             </div>
-            <img id="qr_code" class="w-100" style="image-rendering: pixelated;" src="" alt="QR code" />
+            <img id="qr_code" class="w-100" style="image-rendering: pixelated;" src="" alt="QR code not available" />
```

Comment:
Can you revert this line? It will be always `QR code not available` even the QR code is available.